pyproject.toml

```diff
@@ -10,14 +10,21 @@ fastapi = "^0.45.0"
 uvicorn = "^0.10.8"
```

Comment:
My previous comments about __version__ may not be exactly right in poetry land. Instead from _version.py, you may want to load up this toml file and parse the version that lives here, and have this file be your single source of truth.